cmds/internet/main.go

```diff
@@ -78,15 +82,52 @@ func bootstrap() error {
 		}
 
 		log.Info().Msg("Start network bootstrap")
-		if err := network.Bootstrap(); err != nil {
-			log.Error().Err(err).Msg("fail to boostrap network")
+
+		ifaceConfigs, err := bootstrap.InspectIfaces()
+		if err != nil {
+			log.Error().Err(err).Msg("failed to gather network interfaces configuration")
+			return err
+		}
+
+		zosChild, err := bootstrap.SelectZOS(ifaceConfigs)
+		if err != nil {
+			log.Error().Err(err).Msg("failed to select a valid interface for zos bridge")
+			return err
+		}
+
+		br, err := bootstrap.CreateDefaultBridge(types.DefaultBridge)
+		if err != nil {
 			return err
 		}
 
+		time.Sleep(time.Second) // this is dirty
```

Comment:
Why this is needed?

Reply:
well I'm not sure. I've seen that sometime the call to `netlink.LinkByName` failed to find the interface. 
I suspect it does too fast and the interface has not had the time to be moved back to the host network namespace after the temporary network namespace has been deleted